diff --git a/node_modules/react-leaflet-cluster/lib/index.js b/node_modules/react-leaflet-cluster/lib/index.js
index 8a3f3a7..82b8882 100644
--- a/node_modules/react-leaflet-cluster/lib/index.js
+++ b/node_modules/react-leaflet-cluster/lib/index.js
@@ -10,7 +10,6 @@
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
-require('./assets/MarkerCluster.css');
 
 var MarkerClusterGroup = function (_React$Component) {
   (0, _inherits3.default)(MarkerClusterGroup, _React$Component);